pkg/clusteragent/autoscaling/workload/model/pod_autoscaler.go

```diff
@@ -53,9 +56,15 @@ type PodAutoscalerInternal struct {
 	// (only if owner == remote)
 	settingsTimestamp time.Time
 
-	// scalingValues represents the current target scaling values (retrieved from RC)
+	// isLocalFallbackActive is true if the PodAutoscaler is using local fallback
+	isLocalFallbackActive bool
+
+	// scalingValues represents the main scaling values
 	scalingValues ScalingValues
 
+	// localScalingValues represents the local scaling values
+	localScalingValues ScalingValues
```

Comment:
I'm thinking during a failover, horizontal and vertical are going to come from different sources (one from failover, other from RC), yet we want to build an accurate status of that.
It may be easier to have `scalingValues` become essentially the effective/selected values and having 2 other fields for `main` and another for `failover` or `fallback`.

Reply:
Makes sense, updated!